src/components/member-profile/contribution-type/contributions-type.module.scss

```diff
@@ -39,6 +39,6 @@
 
 @media (max-width: 575.98px) {
   .container {
-    margin: 10px;
```

Comment:
Wouldn't it be better to follow same convention with rest of files, i prefer rem as it has advantage with different resolution of screen than px, I would request you to do more research and come to collective conclusion